with-editor.el

```diff
@@ -176,7 +176,7 @@ please see https://github.com/magit/magit/wiki/Emacsclient."))))
 
 (defcustom with-editor-sleeping-editor "\
 sh -c '\
-echo -e \"WITH-EDITOR: $$ OPEN $0\\037 IN $(pwd)\"; \
+printf \"WITH-EDITOR: $$ OPEN $0\\037 IN $(pwd)\"; \
```

Comment:
Don't we have to append "\n" now?

Reply:
Your are correct; I actually tested it but did not had a look at the output in the shell.
So it was working but the error messages were printed at the end of the WITH-EDITOR lines.

I fixed that and updated the commit.